### ISSUE TITLE
A11y_.NET Core_WPF_DatabindingDemo_ListofProducts_NonTextContrast: The color contrast of the selected list item is less than 3:1.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/wpf#6732]
![6732_A11y_ NET Core_WPF_DatabindingDemo_ListofProducts_NonTextContrast](https://user-images.githubusercontent.com/86710491/176358072-35ab3d7c-b6c1-44b5-810c-344b6af4dfbc.png)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[WPF]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopApp;#Win11;#ColorContrast;#Benchmark;

### Environment Details:
.NET Core
WPF
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Launch VS 2019 Int Preview
2. Navigate to Sample Applications and right click on the DatabindingDemo and click on build.
3. Then right click on DatabindingDemo then Click on Debug-->Start New Instance. 
4. List of products screen should open.
5. Verify all the elements of 'DatabindingDemo_List of Products' screen are accessible and MAS Compliant
6. Turn on Color contrast analyzer and check that The Color contrast ratio of the selected list item is less than 3:1.

### Actual:
The color contrast of the selected list item is less than 3:1.

### Expected:
The color contrast of the selected list item should be more than or equal to 3:1.

### User Impact:
Low vision users won't be able to see the selected list item.

**Comment by dipeshmsft** [comment]
@Truptik12 if I am not mistaken the issue here is that the highlight colours for 1st list item and 2nd list item colour do not have 3:1 ratio right ??

**Comment by Truptik12** [comment]
@dipeshmsft 

The color contrast ratio of the focus indicator on the first list item in the above picture is less than 3:1.
The same focus indicator's ratio is less than 3:1 when it goes to the second list item.

**Comment by singhashish-wpf** [comment]
@Truptik12 
Unable to repro the color contrast bug on our side.
Could you please provide detailed steps and a video so that we can check and modify the colors to adjust the contrast?
Thanks !

https://user-images.githubusercontent.com/82596204/182540542-e129aa8f-7348-47f0-88f2-4a8de3a14691.mp4

**Comment by merriemcgaw** [comment]
@Truptik12 are you using the darker blue line as if it were the focus indicator? That would be incorrect -- you should be using the dashed black line.
@singhashish-wpf - it might be more visible if you thickened the black line and made it solid. Then the focus would really stand out and you wouldn't have concerns from the tester next release. I assume this is just sample app work?

**Comment by Truptik12** [comment]
Verified the bug in the latest environment. The bug does not repro. Hence closing the bug. 
Please find the screenshot below
![image](https://user-images.githubusercontent.com/86710491/183896022-681791a3-acd0-4c31-8840-58b7068b27f7.png)
